GenderPayGap.WebUI/Views/GOVUK/Layouts/template.cshtml

```diff
@@ -191,6 +191,9 @@
                     <li>
                         <a href="/report-concerns">Report Concerns</a>
                     </li>
+                    <li>
+                        <a href="/accessibility-statement">Accessibility Statement</a>

```

Comment:
It would be nicer if this was
```cshtml
<a href="@(Url.Action("AccessibilityStatement", "Home"))">Accessibility statement</a>
```

I know you just copied the ones above, but I prefer the `Url.Action` style because it helps the IDE link the View and the Controler